float32 motEsquerdo
float32 motDireito

